cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -41,6 +41,15 @@ BEGIN
 END;
 $$ LANGUAGE plpgsql;
 
+
+CREATE OR REPLACE FUNCTION set_approx_area_sqkm_to_area_of_geom()
+    RETURNS TRIGGER AS
+$$
+BEGIN
+    NEW.approx_area_sq_km = ST_Area(ST_Transform(new.geom, 3857)) / 1000000;
```

Comment:
Why is this a different SRID than the 4326 used elsewhere?

Reply:
😱 ! This should _absolutely_ be 4326. Thanks!  was running on autopilot.